esmf_regrid/experimental/unstructured_regrid.py

```diff
@@ -35,7 +37,7 @@ def __init__(
         node_coords: :obj:`~numpy.typing.ArrayLike`
             An ``Nx2`` array describing the location of the nodes of the mesh.
             ``node_coords[:,0]`` describes the longitudes in degrees and
-            ``node_coords[:,1]`` describes the latitudes in degrees
+            ``node_coords[:,1]`` describes the latitudes in degrees.
         face_node_connectivity: :obj:`~numpy.typing.ArrayLike`
```

Comment:
in theory, this could become optional, but at the moment MeshInfo is still built around 2D meshes. I think the case of bilinear regridding on non 2D meshes is probably too much of an edge case at the moment to be worth the effort here.